packages/gestalt/src/Masonry/defaultLayout.test.js

```diff
@@ -75,27 +75,46 @@ test('centers grid within the viewport', () => {
     width: 8000,
   });
   expect(layout(items)).toEqual([
-    { top: 0, height: 100, left: 7, width: 236 },
```

Comment:
its unexpected to me that these tests are different. this might just be a git thing, but even looking below at the `centers grid within the viewport` test, the positions are very different